string[] gripper_names
deform_simulator/GripperTrajectory[] trajectories
---
